### DISCUSSION TITLE
[Router v2] Where to import <Head> component from?

### DISCUSSION THREAD

**Discussion by carestad** [expo/router#762] [Q&A] [answered]
The upgrade guide mentions that document titles should be changed by using the `<Head>` component (like in Next), but I don't see where to import it from. My IDE (VScode) does not seem to fin auto-resolve it from any installed packages, so I'm just wondering if anyone knows :slightly_smiling_face:

**Comment by wcastand** [comment, marked as answer]
might help => https://github.com/expo/router/blob/main/apps/sandbox/app/index.tsx
```import Head from "expo-router/head";```

or this one is a different way i guess: https://github.com/expo/router/blob/main/apps/demo/app/%2Bhtml.tsx

**Reply by carestad** [reply]
Woho, yes, that's the one! Thanks a lot :smile:
